src/components/BookCardsPage/BookCardsPage.js

```diff
@@ -4,14 +4,14 @@ import styled from 'styled-components/macro'
 import React, { useState } from 'react'
 import SearchBar from '../SearchBar/SearchBar'
 
-export default function BookCardsPage() {
+export default function BookCardsPage({ onNominate, nominatedBooks }) {
   const [bookmarkedBooks, setBookmarkedBooks] = useState([])
   const [booksShown, setBooksShown] = useState('all')
-  const [isDescriptionExtended, setDescriptionExtended] = useState([])
+  const [descriptionExtended, setDescriptionExtended] = useState([])
   const [userInput, setUserInput] = useState('')
 
   let bookmarkedBooksArray
-  function handleBookmarkClick(currentBook) {
+  function onBookmarkClick(currentBook) {
```

Comment:
Please change that back to `handleBookmarkClick`, as stated above.
```suggestion
  function handleBookmarkClick(currentBook) {
```